Common/src/main/java/com/blamejared/crafttweaker/impl/plugin/crafttweaker/BuiltinCraftTweakerPlugin.java

```diff
@@ -105,6 +109,8 @@ public void registerBracketParsers(final IBracketParserRegistrationHandler handl
         handler.registerParserFor(CraftTweakerConstants.DEFAULT_LOADER_NAME, "tag", new TagBracketHandler(), new IBracketParserRegistrationHandler.DumperData("tag", TagBracketHandler.getDumperData()));
         handler.registerParserFor(CraftTweakerConstants.DEFAULT_LOADER_NAME, "tagmanager", new TagManagerBracketHandler(), new IBracketParserRegistrationHandler.DumperData("tagmanager", TagManagerBracketHandler.getDumperData()));
         
+        handler.registerParserFor(CraftTweakerConstants.TAGS_LOADER_NAME, "tag", new TagBracketHandler(), new IBracketParserRegistrationHandler.DumperData("tag", TagBracketHandler.getDumperData()));
+        handler.registerParserFor(CraftTweakerConstants.TAGS_LOADER_NAME, "tagmanager", new TagManagerBracketHandler(), new IBracketParserRegistrationHandler.DumperData("tagmanager", TagManagerBracketHandler.getDumperData()));
```

Comment:
Should we reuse instances? Also take note that this is likely to lead to duplicate dumpers: register the dumpers in one loader only

Reply:
I don't think we should, it is possible that some bep stores non static data at some point, and it could get confusing